lib/convert.js

```diff
@@ -34,14 +50,16 @@ const opt_string = (value) => {
 
 const opt_tag = (tag) => {
   const retrived_tag = {};
-  const pairs = tag.split(',').map((pair) => pair.split('='));
+  const tag_arr = tag.match(/(\s*([^,]+)".*?"|\s*([^,]+)[^",\s]+)(?=\s*,|\s*$)/g); //same regex as in param_arr
```

Comment:
I wouldn't put this here, this comment's gonna become out of date and then it'll be a lie